frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.tsx

```diff
@@ -78,6 +79,11 @@ function ModelDetailPage({
 }: Props) {
   const [hasFetchedTableMetadata, setHasFetchedTableMetadata] = useState(false);
 
+  const database = model.database()?.getPlainObject();
```

Comment:
Please note that `getPlainObject` returns a **normalized** enity without hydrating referenced entities. E.g. it would be something like this:
```
{
  id: 1,
  name: "...",
  tables: [1, 2, 3] // not `Table[]` as defined by the type
}
```

Depending on how you use that type things might go very wrong.